src/extensions/DatatypeTrait.php

```diff
@@ -71,4 +73,41 @@ public function getDatatypes() {
 
         return !empty($this->data_types) ? $this->data_types : [];
     }
+
+    /**
+     * Check data type of titles / first row for auto detecting if this could be
+     * a heading line.
+     *
+     * Requires PHP >= 5.5
+     *
+     * @access public
+     *
+     * @uses   DatatypeEnum::getValidTypeFromSample
+     *
+     * @return bool
+     */
+    public function autoDetectFileHasHeading(){
+        if (empty($this->data)){
+            throw new \UnexpectedValueException('No data set yet.');
+        }
+
+        if ($this->heading){
+            $firstRow = $this->titles;
+        } else {
+            $firstRow = $this->data[0];
+        }
+
+        $firstRow = array_filter($firstRow);
+        if (empty($firstRow)){
+            return false;
+        }
+
+        $firstRowDatatype = array_map('ParseCsv\enums\DatatypeEnum::getValidTypeFromSample', $firstRow);
```

Comment:
That line should read something like `array_map(ParseCsv\enums\DatatypeEnum::class . '::getValidTypeFromSample', $firstRow)` to make sure the file is autoloaded and to allow the IDE to complain if class name or namespace change in the future.

Reply:
Okay, I have changed it. Thanks.